docker-compose.yml

```diff
@@ -8,9 +8,10 @@ services:
         - http_proxy
         - https_proxy
         - no_proxy
-    image: hmctspublic.azurecr.io/spring-boot/template
+    image: hmcts/civil-rtl-export
     environment:
       # these environment variables are used by java-logging library
+      - JF_DB_HOST=civil-jf-database
```

Comment:
Not sure if we need this line.  Make all the other changes, build the application and test that it works, then remove this line and try it again.  If it works OK without it then remove it.

Reply:
The line is needed.  The value needs to be set to the name of the database service which in this case is civil-rtl-export-database.